src/developer-documentation/ibc/ibc.md

```diff
@@ -0,0 +1,999 @@
+
+# 1 ibc infomation
```

Comment:
Can we use capital letters where possible for general terms like IBC, NFT, etc?

Reply:
sure